overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -54692,31 +54808,43 @@
               "id:8": "gregtech:computer_casing"
             },
             "4:10": {
-              "Count:3": 4,
-              "Damage:2": 1413,
+              "Count:3": 1,
+              "Damage:2": 1410,
               "OreDict:8": "",
               "id:8": "gregtech:machine"
-            },
-            "5:10": {
+            }
+          },
+          "taskID:8": "bq_standard:retrieval"
+        },
```

Comment:
Either Task Logic OR or combine into single task.

Reply:
I feel like OR is the correct choice here, since the player shouldn't be forced to use the configuration recommended.